website/content/labs/lab1.md

```diff
@@ -222,45 +239,45 @@ So, if you run `git add -u` and create a commit:
+If you see `nothing to commit, working directory clean`, that means that there are no changes in your local repository since the last commit you created (and, additionally, the above output also tells us that all our commits have also been uploaded to GitHub)
 
 ## Working from multiple locations
 
-So far, you have a local repository in your CS home directory, which you have been uploading to the GitLab server using the `git push` command. However, if you work from multiple locations (e.g., on a CS machine but also from your laptop), you will need to be able to create a local repository in those locations too. You can do this by running the `git clone` command (don't run this command just yet):
+So far, you have a local repository in your CS home directory, which you have been uploading to GitHub using the `git push` command. However, if you work from multiple locations (e.g., on a CS machine but also from your laptop), you will need to be able to create a local repository in those locations too. You can do this by running the `git clone` command (don't run this command just yet):
 
-    git clone https://mit.cs.uchicago.edu/cmsc22000-spr-19/username.git
+    git clone https://github.com/cmsc22000-labs/2020-lab1-username.git
 
-This will create a local repository that "clones" the contents of the repository in the GitLab server. If you have a laptop with you (and have access to Git on that laptop), try running `git clone` there. If not, try creating a clone in a different directory on the CS machine you are on. For example:
+This will create a local repository that "clones" the version of the repository that is currently stored on GitHub. Let's do this in a different directory on the machine you are on. For example:
 
     mkdir -p /tmp/$USER/cs220
     cd /tmp/$USER/cs220
-    git clone https://mit.cs.uchicago.edu/cmsc22000-spr-19/username.git
+    git clone https://github.com/cmsc22000-labs/2020-lab1-username.git
 
-Make sure to replace `username` with your CNetID!
+Make sure to replace `username` with your GitHub username!
 
 Take into account that, when you run `git clone`, the repository is not cloned *into* the current directory. Instead, a *new* directory (with the same name as the repository) will be created in the current directory, and you will need to `cd` into it to use Git commands for that repository.
 
-Now, in the local repository in your home directory, add a line to `test.txt` with the text `One more change!`. Create a commit for that change and push it to GitLab (you should know how to do this by now, but make sure to ask for help if you're unsure of how to proceed).
+Now, in the local repository in your home directory, add a line to `test.txt` with the text `One more change!`. Create a commit for that change and push it to GitHub (you should know how to do this by now, but make sure to ask for help if you're unsure of how to proceed).
```

Comment:
As these labs used to be in-person, the part where it says "make sure to ask for help" made more sense, but as we've moved onto only using Zoom and Slack, I think we should expand upon where they should ask for help if they are stuck with the lab.

Now that I think of it, I'm not even sure where they should ask for help about an issue like this. I assume #general on Slack? Or DMing one of us?

Reply:
I feel like a dedicated slack channel with rigid threading etiquette would be the right move here

Reply:
As far as Slack support is concerned, there will be a #labs channel for questions about the labs (where questions will have to be threaded). We're also going to have office hours over Zoom, but we're still trying to figure out how those will work exactly. Once we figure all this out, I'll be adding a "Getting Help" section to the syllabus (but will also explain how the #labs channel will work when I release the lab)